### PR TITLE
viewvc: set maximum macOS

### PR DESCRIPTION
**Pull request by iMichka**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/viewvc.rb [hunk 1]
```diff
@@ -14,7 +14,7 @@ class Viewvc < Formula
     sha256 cellar: :any_skip_relocation, high_sierra:   "19c07a79667814ccb1b14b6214a3d5fcca65ec31381e6e46a5db3ac3f72fc2d4"
   end
 
-  deprecate! date: "2022-10-27", because: "has no python 3 support"
+  deprecate! date: "2023-04-17", because: "has no python 3 support"
```
Comment (ZhongRuoyu) [inline Formula/viewvc.rb:17]:
```suggestion
  disable! date: "2023-04-17", because: "has no python 3 support"
```

**Review by ZhongRuoyu — COMMENTED** [review]
Actually, upstream will provide support for Python 3 starting from their upcoming 1.3.x release series: https://github.com/viewvc/viewvc#requirements

So, maybe we can keep it as it is?

**Review by carlocab — DISMISSED** [review]

**Comment by SMillerDev** [comment]
I'd rather reenable if they ever release that

**Comment by carlocab** [comment]
We already have a workflow that disables deprecated formulae after a year, we can leave this till then I think. No real reason to rush this?

**Comment by SMillerDev** [comment]
> We already have a workflow that disables deprecated formulae after a year, we can leave this till then I think. No real reason to rush this?

I think that workflow is a neural net it's developers called @iMichka, but we could wait.

**Comment by iMichka** [comment]
> We already have a workflow that disables deprecated formulae after a year

We have a workflow that deletes disabled formulae after a year. Going from deprecation to disabled is a manual thing I do from time to time, based on a script written by @SMillerDev (I never had the courage to make it a proper brew command but maybe one day).

What is also happening is that I am working through the list of formulae that don't have a Ventura bottle: https://github.com/Homebrew/homebrew-core/issues/123214. I had this deprecated formulae in the list, that's all.

Our rules say we can disable them after 3 months of deprecation, so I thought I would just do it. After being disabled there is still one full year before the final removal.

File: Formula/viewvc.rb [hunk 2]
```diff
@@ -14,7 +14,7 @@ class Viewvc < Formula
     sha256 cellar: :any_skip_relocation, high_sierra:   "19c07a79667814ccb1b14b6214a3d5fcca65ec31381e6e46a5db3ac3f72fc2d4"
   end
 
-  deprecate! date: "2022-10-27", because: "has no python 3 support"
+  disable! date: "2023-04-17", because: "has no python 3 support"
 
   depends_on :macos # Due to Python 2 (https://github.com/viewvc/viewvc/issues/138)
```
Comment (carlocab) [inline Formula/viewvc.rb:19]: I suggest making this `:maximum_macos` and let it die a slow death instead.

Comment (iMichka) [reply Formula/viewvc.rb:19]: Done

**Review by p-linnane — APPROVED** [review]

**Pull request merged**